app/gui/src/dashboard/layouts/AssetPanel/components/AssetProperties.tsx

```diff
@@ -166,6 +175,7 @@ function AssetPropertiesInternal(props: AssetPropertiesInternalProps) {
     editDescriptionMutation.variables?.[0] === item.id ?
       (editDescriptionMutation.variables[1].description ?? item.description)
     : item.description
+  const ownerPermission = permissions.tryGetOwnerPermission(item)
```

Comment:
user is always an owner now, this does nothing

Reply:
even for team assets? is there a heuristic we can use, or should we remove this for now?

Reply:
Good question, please ask @PabloBuchu